qa/tests/src/org/jboss/jbossts/qa/junit/testgroup/TestGroup_crashrecovery12.java

```diff
@@ -124,4 +125,12 @@ public String getTestGroupName()
 		outcome0.start("CR12_07.log", "no");
 		outcome0.waitFor();
 	}
+
+	private void suspend(long millis) {
+		try {
+			Thread.sleep(millis);
+		} catch (InterruptedException e) {
+			System.out.println("Suspend interrupted: " + e.getMessage());
```

Comment:
Normally there is a reason the thread has been interrupted so you should either re-throw the  InterruptedException or re-enable the interrupt status with:

`Thread.currentThread().interrupt() `